### PR TITLE
Query : PeerVersion

### PR DESCRIPTION
**Pull request by parv0888**
## Purpose
Adds support for `PeerVersion`

## Changes
Adds support for `PeerVersion`

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

### CODE REVIEW

**Review by EmilLa1 — CHANGES_REQUESTED** [review]
While I empathize with the decision of adding the `Semver`  package.  Unfortunately  currently it does not look like it is properly maintained, see e.g.:  https://github.com/vdurmont/semver4j/issues/57

I suggest we just store the node version in a `String` for now.

**Comment by parv0888** [comment]
> 
Sure @EmilLa1  - Rust SDK was using a a similar approach to return a parsed semantic version. 
Thanks. Its done

**Comment by abizjak** [comment]
> > 
> 
> Sure @EmilLa1 - Rust SDK was using a a similar approach to return a parsed semantic version. Thanks. Its done

I think the point is that the dependency was not maintained, not that we should not be parsing the response.

The Rust SDK uses the canonical ecosystem semver parsing library that is used, among others, but the compiler toolchain itself.

**Comment by parv0888** [comment]
> > > 
> > 
> > 
> > Sure @EmilLa1 - Rust SDK was using a a similar approach to return a parsed semantic version. Thanks. Its done
> 
> I think the point is that the dependency was not maintained, not that we should not be parsing the response.
> 
> The Rust SDK uses the canonical ecosystem semver parsing library that is used, among others, but the compiler toolchain itself.

@EmilLa1 , @abizjak  - Can you please suggest a library. I will use that.

**Comment by EmilLa1** [comment]
Looks like there is a maintained fork semver4j. 

https://github.com/semver4j/semver4j

I would suggest going with that.

**Comment by parv0888** [comment]
> @parv0888 Looks like there is a maintained semver4j fork .
> 
> https://github.com/semver4j/semver4j
> 
> I would suggest going with that.

Thanks @EmilLa1 
its been done

File: README.md [hunk 1]
```diff
@@ -336,6 +342,11 @@ val uptime = client.getUptime();
 val sentPackets = client.getTotalSent();
 ```
 
+#### getVersion
+```java
+String version = client.getVersion();
```
Comment (EmilLa1) [inline README.md:347]:
```suggestion
SemVer version = client.getVersion();
```

File: README.md [hunk 2]
```diff
@@ -195,6 +195,12 @@ long getTotalSent()
 ```
 Retrives total number of packets sent from the node.
 
+- `getVersion`
+```java
+String getVersion()
```
Comment (EmilLa1) [inline README.md:200]:
```suggestion
SemVer getVersion()
```

File: README.md (hunk 1 above)
Comment (parv0888) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (parv0888) [reply README.md]: done

**Review by EmilLa1 — APPROVED** [review]

**Comment by EmilLa1** [comment]
Please rebase and we can merge it.

**Comment by parv0888** [comment]
> Please rebase and we can merge it.

thanks, done

**Pull request merged**